app/src/main/java/com/globant/marvelmvvm/util/Constants.kt

```diff
@@ -0,0 +1,9 @@
+package com.globant.marvelmvvm.util
+
+class Constants {
+    companion object{
+        const val EMPTY_STRING = ""
```

Comment:
The correct use of an a Contants class is to define constants into the objet...but if you put it into the current componion obj it lose sense. So I recommend to put all the variable into the class removing the componion.

Reply:
You are right, I should not use that name but I will use the approach jaja, thanks!